### ISSUE TITLE
my_cube.dispersion method available but shouldn't be

### ISSUE THREAD

**Issue by KateRowlands** [sdss/marvin#546]
```
my_cube = marvin.tools.Cube('7443-12703')
my_cube.dispersion 
```

is available as a method, but it shouldn't be as the dispersion cube is accessed a different way. It produces the following crash:

```
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/extern/marvin_brain/python/brain/api/api.py in _checkResponse(self, response)
    197         try:
--> 198             isbad = response.raise_for_status()
    199         except requests.HTTPError as http:

//anaconda/envs/msaviz/lib/python3.6/site-packages/requests/models.py in raise_for_status(self)
    934         if http_error_msg:
--> 935             raise HTTPError(http_error_msg, response=self)
    936 

HTTPError: 422 Client Error: UNPROCESSABLE ENTITY for url: https://lore.sdss.utah.edu/public/marvin/api/cubes/7443-12703/extensions/disp/

During handling of the above exception, another exception occurred:

BrainError                                Traceback (most recent call last)
//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/tools/cube.py in _get_extension_data(self, name, ext)
    405                                cube_extension=model.fits_extension(ext).lower()),
--> 406                     params=params)
    407             except Exception as ee:

//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/tools/core.py in _toolInteraction(self, url, params)
    131         params = params or {'release': self._release}
--> 132         return marvin.api.api.Interaction(url, params=params)
    133 

//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/extern/marvin_brain/python/brain/api/api.py in __init__(self, route, params, request_type, auth, timeout, headers, stream, datastream, send)
     53         if self.url and send:
---> 54             self._sendRequest(request_type)
     55         elif not self.url and send:

//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/extern/marvin_brain/python/brain/api/api.py in _sendRequest(self, request_type)
    289             # Check the response if it's good
--> 290             self._checkResponse(self._response)
    291 

//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/extern/marvin_brain/python/brain/api/api.py in _checkResponse(self, response)
    220                 self._closeRequestSession()
--> 221                 raise BrainError('Requests Http Status Error: {0}\nValidation Errors:\n{1}'.format(http, json_data))
    222             else:

BrainError: Requests Http Status Error: 422 Client Error: UNPROCESSABLE ENTITY for url: https://lore.sdss.utah.edu/public/marvin/api/cubes/7443-12703/extensions/disp/
Validation Errors:
{'validation_errors': {'cube_extension': ['Not a valid choice.']}}

During handling of the above exception, another exception occurred:

MarvinError                               Traceback (most recent call last)
<ipython-input-53-f02f33ad3046> in <module>()
----> 1 my_cube.dispersion

//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/tools/cube.py in dispersion(self)
    314 
    315         if self._dispersion is None:
--> 316             self._dispersion = self._get_datacube('dispersion')
    317 
    318         return self._dispersion

//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/tools/cube.py in _get_datacube(self, name)
    258 
    259         model = self.datamodel.datacubes[name]
--> 260         cube_data = self._get_extension_data(name)
    261 
    262         if cube_data is None:

//anaconda/envs/msaviz/lib/python3.6/site-packages/marvin/tools/cube.py in _get_extension_data(self, name, ext)
    407             except Exception as ee:
    408                 raise MarvinError('found a problem when checking if remote cube '
--> 409                                   'exists: {0}'.format(str(ee)))
    410 
    411             data = response.getData()

MarvinError: found a problem when checking if remote cube exists: Requests Http Status Error: 422 Client Error: UNPROCESSABLE ENTITY for url: https://lore.sdss.utah.edu/public/marvin/api/cubes/7443-12703/extensions/disp/
Validation Errors:
{'validation_errors': {'cube_extension': ['Not a valid choice.']}}.
You can submit this error to Marvin GitHub Issues (https://github.com/sdss/marvin/issues/new).
Fill out a subject and some text describing the error that just occurred.
If able, copy and paste the full traceback information into the issue as well.
```

If the map is not available then the method shouldn't be an available option.

**Comment by KateRowlands** [comment]
Also, the name 'dispersion' is confusing to me, it makes me think of velocity dispersion. It owuld be better to change dispersion to LSF.

**Comment by albireox** [comment]
Hi @KateRowlands. Thanks for reporting and sorry for the delay. What MPL and configuration are you using? Using MPL-6 or 7 I'm not able to reproduce it.

What do you mean the dispersion of the cube is accessed differently? I'm not opposed to call this lsf if people agree. We kept the name from the FITS header.

**Comment by KateRowlands** [comment]
I'm using DR15, default config when I get the error.

I thought there was another way to access the dispersion but I can't find what it is, I think I was mistaken.

**Comment by albireox** [comment]
Got it. `disp` and `predisp` were not included in the validator for the API URL. It's fixed in 50ed84e2b0fce49a231ba2efc6066fd44167f5c0. Thanks!
